src/lib/ThumbnailsSidebar.js

```diff
@@ -316,16 +317,16 @@ class ThumbnailsSidebar {
                 // landscape
                 if (curPageRatio < this.pageRatio) {
                     // Set the canvas height to that of the thumbnail max height
-                    canvas.height = Math.ceil(DEFAULT_THUMBNAILS_SIDEBAR_WIDTH / this.pageRatio);
+                    canvas.height = Math.floor(HIGHER_RES_WIDTH / this.pageRatio);
```

Comment:
What was the reason for moving to `Math.floor` vs `Math.ceil`? Were we seeing overflow previously?

Reply:
Was trying to fix an issue where an extra line of whitespace was getting generated as part of the image